releases.md

```diff
@@ -0,0 +1,38 @@
+# Proposed Revised Repo Model and Release Plan
+
+The Node.js TSC has recently been discussing and hammering out the details
+for a revised general release plan. The fundamental details of the plan are:
+
+<img src="https://camo.githubusercontent.com/675f052419c56a583c19644d4da30cc0ff4e02bb/68747470733a2f2f636c6475702e636f6d2f6c4d48746245334e72782d3330303078333030302e706e67"/>
+
+The `master` branch remains as the target of primary development. The majority of commits generally land there. The overall goal is to keep master as stable as possible, with periodic, numbered *alpha* releases being cut at fairly regular intervals.
+
+Every six months, starting from the when the *first* LTS release is cut from the fully converged repo (nodejs/node), the *semver-major* is bumped in master and a new *stable branch* is created for the previous major. In other words, if `master` is currently on `v3.1.3` and is being bumped to `v4.0.0`, the new stable branch is cut off `v3.1.3`. Then, when `master` transitions to `v5.0.0` six months later, a new stable is cut off `v4.x.x`.
```

Comment:
As @Fishrock123 said, as soon as master reaches X.0.0 a branch will be cut, and future releases of stable are cut from that branch.

Likewise, LTS has no need to cut a branch. It will simply inherit the stable branch after its last release is done.